pkg/controller/jobs/kubeflow/jobs/pytorchjob/pytorchjob_controller.go

```diff
@@ -57,6 +60,10 @@ var NewReconciler = jobframework.NewGenericReconciler(func() jobframework.Generi
 	return &kubeflowjob.KubeflowJob{KFJobControl: (*JobControl)(&kftraining.PyTorchJob{})}
 }, nil)
 
+func isPyTorchJob(owner *metav1.OwnerReference) bool {
+	return owner.Kind == "PyTorchJob" && strings.HasPrefix(owner.APIVersion, "kubeflow.org/v2")
```

Comment:
```suggestion
	return owner.Kind == "PyTorchJob" && strings.HasPrefix(owner.APIVersion, "kubeflow.org")
```

the `v2` requirement is specific to MPIJob